texar/torch/modules/encoders/distilbert_encoder.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+DistilBERT encoder.
+"""
+
+from typing import Optional
+
+import torch
+
+from texar.torch.modules.encoders.bert_encoder import BERTEncoder
+from texar.torch.modules.pretrained.pretrained_distilbert import \
+    PretrainedDistilBERTMixin
+
+__all__ = [
+    "DistilBERTEncoder",
+]
+
+
+class DistilBERTEncoder(PretrainedDistilBERTMixin, BERTEncoder):  # type: ignore
+    r"""DistilBERT Transformer for encoding sequences.
+
```

Comment:
Add a sentence: "Please see :class:`~texar.torch.modules.PretrainedDistilBERTaMixin` for a brief description of DistilBERT".

All other pretrained modules should add such a sentence